python/lsst/ci/prepare.py

```diff
@@ -264,6 +281,28 @@ def fetch(self, product):
 
         """
 
+        for i in range(self.retries - 1):
+            try:
+                return self._fetch(product)
+            except (GitError, RemoteError, OSError) as e:
+                print('<error>', file=self.out)
```

Comment:
Is everything in this package using prints and not `logging`? (e.g., `logger.exception()`)

Reply:
That's correct, there is no existing use of `logging`.  I'm a little hesitant to convert it over after all the grief we've had with needing to manually flush output.